templates/saturnbase-gpu/Dockerfile

```diff
@@ -66,7 +66,7 @@ COPY install-jupyter.bash /tmp/install-jupyter.bash
 COPY jupyter.yml /tmp/jupyter.yml
 RUN bash /tmp/install-jupyter.bash
 
-RUN ${CONDA_DIR}/bin/jupyter serverextension enable --py nbserverproxy --sys-prefix
+RUN ${CONDA_DIR}/bin/jupyter serverextension enable jupyter-server-proxy --sys-prefix
```

Comment:
Based on the [readme](https://github.com/jupyterhub/jupyter-server-proxy) it seems like we should use the new method of installing like we do for `jupyterlab_bokeh` a few lines down.

Reply:
> Based on the [readme](https://github.com/jupyterhub/jupyter-server-proxy) it seems like we should use the new method of installing like we do for `jupyterlab_bokeh` a few lines down.

I think that's only necessary/useful for the UI around jupyter server proxy apps?  To get the backend to do the proxying I think we still have to enable the serverextension?  labextensions only build JS iirc, and there is definitely backend python at play here.

Reply:
@jsignell my comment assumes you were talking about 

```
jupyter labextension install @jupyterlab/server-proxy
 
```

Reply:
ok

Reply:
ok makes sense. Ignore me @andredias :facepalm: